app/models/ingredient.rb

```diff
@@ -3,7 +3,18 @@
 class Ingredient < ApplicationRecord
   validates :name, uniqueness: true
 
+  MEASURES = %w[cup oz ml fl oz
```

Comment:
I'd specter both `MEASURES` and `CATEGORIES` to be unidimensional collection(i.e ['value1', 'value2'])
But instead they are a multidimensional one.
I don't know why do you think the hay to be by-dimensional, but probably that operation has to be done somewhere else.
Let's make them a single array

Reply:
This is being removed in the latest commit.